gitutil.py

```diff
@@ -137,7 +137,7 @@ def clone_repository(
         github_repo_path: str
     ):
         url = url_with_credentials(github_cfg, github_repo_path)
-        git.Git(to_path).clone('--quiet', url)
+        git.Git().clone('--quiet', url, to_path)
```

Comment:
you need to adjust all usages (currently one, if I properly recall). Please grep..